### PR TITLE
Feature/14928 integrate answer service

### PR DESCRIPTION
**Pull request by GeorgeChatzigiannis**
- Answer service integration with check component
- Added tests

### CODE REVIEW

**Comment by jon-shipley** [comment]
@George - please resolve the conflicts so I can review

**Comment by GeorgeChatzigiannis** [comment]
@jon-shipley resolved

File: pupil-spa/src/app/check/check.component.ts [hunk 1]
```diff
@@ -48,11 +49,15 @@ export class CheckComponent implements OnInit {
 
   manualSubmitHandler(answer: string) {
     // console.log(`check.component: manualSubmitHandler(): ${answer}`);
+    const answerSet = { factor1: this.question.factor1, factor2: this.question.factor2, answer: parseInt(answer, 10) };
```
Comment (jon-shipley) [inline pupil-spa/src/app/check/check.component.ts]: I think should be a string here as we need to record leading zero's `in the answer`.

Comment (jon-shipley) [inline pupil-spa/src/app/check/check.component.ts]: It would also be good to store the sequence number too.  Is this Q1 or Q9?

Comment (GeorgeChatzigiannis) [inline pupil-spa/src/app/check/check.component.ts]: @jon-shipley modified the answer to be a string

Comment (GeorgeChatzigiannis) [inline pupil-spa/src/app/check/check.component.ts]: @jon-shipley is that required? I would assume you will always have the answers in the proper order, hence you will be able to distinguish the sequence.

**Pull request merged**